test/test_oneview_volume_facts.py

```diff
@@ -1,26 +1,10 @@
-#!/usr/bin/python
-# -*- coding: utf-8 -*-
-###
-# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# You may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-###
-
-from ansible.compat.tests import unittest
-from oneview_module_loader import VolumeFactsModule
-from hpe_test_utils import FactsParamsTestCase
+# Copyright: (c) 2016-2017 Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
@ricardogpsf Do we need to add GPU 3.0 or continue using Apache 2.0.

Reply:
no! Thanks for this review, I'll change back to the Apache license.
We are using Apache license inside this repo, just when move the code to the ansible core that we need change to the license of ansible git repo.